### PR TITLE
Add coverage support

### LINKED ISSUE #32: Add code coverage

### PR DESCRIPTION
**Pull request by jelly**
Fixes #32

### CODE REVIEW

**Comment by jelly** [comment]
Nevermind, this can't work due to the rpm hardcoding `ESLINT=0 NODE_ENV=production make`

**Comment by martinpitt** [comment]
I linked this to issue #32

File: test/run [hunk 1]
```diff
@@ -8,6 +8,12 @@ TEST_SCENARIO="${TEST_SCENARIO:-}"
 [ "${TEST_SCENARIO}" = "${TEST_SCENARIO##firefox}" ] || export TEST_BROWSER=firefox
 export RUN_TESTS_OPTIONS=--track-naughties
 
+TEST_SCENARIO=${TEST_SCENARIO:-}
+
```
Comment (martinpitt) [inline test/run:12]: This line is redundant and confusing, line 7 already does that.

**Review by martinpitt — CHANGES_REQUESTED** [review]
Thanks! LGTM except for what I take is a copy pasta.

File: test/run (hunk 1 above)
Comment (jelly) [reply test/run]: Woops, fixed and added the issue to the commit.

**Review by martinpitt — APPROVED** [review]
wat leuk, dank je!

**Pull request merged**